### DISCUSSION TITLE
How to schedule thousands of jobs in bulk using JDBC batch?

### DISCUSSION THREAD

**Discussion by v0k01** [kagkarlsson/db-scheduler#427] [General]
I was evaluating db-scheduler. Our use case requires generating around **20000** jobs within 1 minute and schedule them. Jobs should get processed in around 15 minutes (each job takes between 5-10 seconds). 

However I am stuck at first step itself i.e. how to schedule 20000 jobs within a minute since each call to `schedulerClient.schedule` does a DB insert and they are not batched. So question is how do I schedule thousands of jobs within few seconds or a minute ?

**Comment by kagkarlsson** [comment]
Yeah currently you have to do it one by one. It should be possible to add support for scheduling batches in the client.

**Reply by kagkarlsson** [reply]
https://github.com/kagkarlsson/db-scheduler/issues/428
